src/parser/parser.h

```diff
@@ -24,23 +24,26 @@
 #include <list>
 #include <cassert>
 
-#include "parser/input.h"
-#include "parser/parser_exception.h"
+#include "api/cvc4cpp.h"
 #include "expr/expr.h"
-#include "expr/symbol_table.h"
-#include "expr/kind.h"
 #include "expr/expr_stream.h"
+#include "expr/kind.h"
+#include "expr/symbol_table.h"
+#include "parser/input.h"
+#include "parser/parser_exception.h"
 #include "util/unsafe_interrupt_exception.h"
 
 namespace CVC4 {
 
 // Forward declarations
 class BooleanType;
-class ExprManager;
 class Command;
 class FunctionType;
 class Type;
 class ResourceManager;
+namespace api {
+class Solver;
+}
```

Comment:
We shouldn't need this forward declaration since we include `api/cvc4cpp.h`.

Reply:
We actually don't need the include, removed.